docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,425 @@
+# Shapella mainnet parameters
+
+## Mainnet addresses
+
+```python
+lidoLocator = 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb
+legacyOracle = 0x442af784A788A5bd6F42A01Ebe9F287a871243fb  # was `LidoOracle` before
+
+# See https://docs.lido.fi/deployed-contracts/
+withdrawalsVault = 0xB9D7934878B5FB9610B3fE8A5e441e8fad7E293f
+
+# Same as withdrawalsVault with 0x01 type prefix
+# also see https://etherscan.io/address/0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84#readProxyContract#F23
+withdrawalsCredentials = 0x010000000000000000000000b9d7934878b5fb9610b3fe8a5e441e8fad7e293f
+```
+
+## NodeOperatorsRegistry
+
+```python
+# See https://research.lido.fi/t/lido-validator-exits-policy-draft-for-discussion/3864/5
+stuckPenaltyDelay = 432000  # 5 days as seconds
+```
+
+## OracleDaemonConfig
+
+```python
+# See https://research.lido.fi/t/withdrawals-for-lido-on-ethereum-bunker-mode-design-and-implementation/3890
+NORMALIZED_CL_REWARD_PER_EPOCH=64  # factor
+NORMALIZED_CL_REWARD_MISTAKE_RATE_BP=1000  # 10%
+REBASE_CHECK_NEAREST_EPOCH_DISTANCE=1
+REBASE_CHECK_DISTANT_EPOCH_DISTANCE=23  # 10% of AO 225 epochs frame
+
+# See https://research.lido.fi/t/lido-validator-exits-policy-draft-for-discussion/3864/6
+VALIDATOR_DELAYED_TIMEOUT_IN_SLOTS=7200  # 1 day
+VALIDATOR_DELINQUENT_TIMEOUT_IN_SLOTS=28800  # 4 days
+NODE_OPERATOR_NETWORK_PENETRATION_THRESHOLD_BP=100  # 1% network penetration for a single NO
+
+# Time period of historical observations used for prediction of the rewards amount 
+PREDICTION_DURATION_IN_SLOTS=50400  # 7 days
+
+# Max time upon a withdrawal request won't be finalized in case of negative rebase
+FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT=1350  # 6 days (twice min governance response time - 3 days voting duration)
+```
+
+## OracleReportSanityChecker
+
+```python
+# Sanity limit on the amount of deposits: not more than approximately deposit every 2nd block
+# https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L221-L232
+churnValidatorsPerDayLimit = 20000
+
+# Taken from current oracle limit https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb#readProxyContract#F8
+# See https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L47-L50
+oneOffCLBalanceDecreaseBPLimit = 500  # 5%
+
+# See https://research.lido.fi/t/increasing-max-apr-sanity-check-for-oracle-lido-report/3205
+annualBalanceIncreaseBPLimit = 1000  # 10%
+
+# See https://research.lido.fi/t/increasing-max-apr-sanity-check-for-oracle-lido-report/3205
```

Comment:
please change to https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L647-L672